pkg/parser/task/command/cache.go

```diff
@@ -38,21 +38,23 @@ func NewCacheCommand(mergedEnv map[string]string, boolevator *boolevator.Booleva
 		return nil
 	})
 
-	folderSchema := schema.String("Path of a folder to cache.")
+	folderSchema := schema.StringOrListOfStrings("Path of a folder or a list of folders to cache.")
 	cache.RequiredField(nameable.NewSimpleNameable("folder"), folderSchema, func(node *node.Node) error {
```

Comment:
Should it be a separate field `folders`? The question is how to make sure that either `folder` or `folders` is set. 🤔  Maybe make them optional and validate that `Fodlers` is not empty?

Reply:
> Should it be a separate field `folders`?

We already have the `script` with the same single/multi semantics, why not continue the trend and there will be no issue with require/optional validation problems?

Reply:
With a script you kind of making it multiline even though it's a YAML list. IMO it's just weird to see

```yaml
cache:
  folder:
    - cache/foo
    - cache/bar
    - cache/baz
  ```

Reply:
> IMO it's just weird to see

Looks similar to `script` to me:

```yaml
task:
  script:
    - make test
    - make build
```

The user doesn't know about the multiline internals anyway.

But I can split it into separate fields if you still think it should be done this way.

Reply:
Oh, I think I've got what you mean by `script` being multiline. I'll add another field then.

Reply:
Implemented in https://github.com/cirruslabs/cirrus-cli/pull/421/commits/575a99371b50739291b87bbcc29f1fd0528d6ae0.